squid/datadog/squid/squid.py

```diff
@@ -0,0 +1,130 @@
+# (C) Datadog, Inc. 2010-2016
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+# stdlib
+
+# 3rd party
+import requests
+
+# project
+from checks import AgentCheck
+
+EVENT_TYPE = SOURCE_TYPE_NAME = 'squid'
+
+METRIC_PREFIX = "squid"
+SERVICE_CHECK = "squid.can_connect"
+SQUID_COUNTERS = [
```

Comment:
These are all cachemanager metrics. I believe Squid may have some other metrics - should we append `cachemgr` to the metric namespace?

Reply:
Oh I didn't know there were other metrics than the `cachemanager` ones. I can absolutely add that to the prefix !